recipe/conda_build_config.yaml

```diff
@@ -1,8 +1,14 @@
 python:                        # [win]
   - 3.8                        # [win]
-c_compiler:                    # [win]  
+c_compiler:                    # [win]
   - vs2017                     # [win]
 cxx_compiler:                  # [win]
   - vs2017                     # [win]
-vc:                            # [win]   
-  - 14.1                       # [win] 
+vc:                            # [win]
+  - 14.1                       # [win]
+fortran_compiler_version:      # [win]
+  - 2019.0.0                   # [win]
+nodejs:                        # [win]
+  - 8.12                       # [win]
+geos:                          # [win]
```

Comment:
Why this one?

Reply:
@mingwandroid can you answer those two comments please?  Just want to be sure about them before merging.  Thanks